workbench-core/authorization/src/dynamicAuthorization/dynamicAuthorizationService.test.ts

```diff
@@ -350,24 +350,61 @@ describe('DynamicAuthorizationService', () => {
   });
 
   describe('addUserToGroup', () => {
-    it('returns userID and groupID when user was successfully added to the group', async () => {
-      mockGroupManagementPlugin.addUserToGroup = jest.fn().mockResolvedValue({ data: { userId, groupId } });
+    beforeAll(() => {
+      auditAction = 'addUserToGroup';
+    });
 
-      const { data } = await dynamicAuthzService.addUserToGroup({
-        groupId,
+    afterEach(jest.resetAllMocks);
+
+    it('returns userID and groupID when user was successfully added to group', async () => {
+      const mockReturnValue = { data: { userId, groupId } };
+      mockGroupManagementPlugin.addUserToGroup = jest.fn().mockResolvedValue(mockReturnValue);
+
+      const requestBody = {
         userId,
+        groupId,
         authenticatedUser: mockUser
-      });
+      };
+
+      const { data } = await dynamicAuthzService.addUserToGroup(requestBody);
```

Comment:
```suggestion
      const response = await dynamicAuthzService.addUserToGroup(requestBody);
```

Reply:
lets test the full response so we can know if the response object changes without updating the tests